spring-boot-project/spring-boot-tools/spring-boot-test-support-docker/src/main/java/org/springframework/boot/testsupport/container/TestImage.java

```diff
@@ -272,7 +272,12 @@ public enum TestImage {
 	/**
 	 * A container image suitable for testing Redis via Bitnami.
 	 */
-	BITNAMI_REDIS("bitnami/redis", "7.2.4");
+	BITNAMI_REDIS("bitnami/redis", "7.2.4"),
+
+	/**
+	 * A container image suitable for testing Hazelcast.
+	 */
+	HAZELCAST("hazelcast/hazelcast", "5.5.0-slim", () -> HazelcastContainer.class);
```

Comment:
The values in that enum should be sorted alphabetically. (All the bitnami images are at the bottom).